src/components/BookForm.js

```diff
@@ -29,13 +30,12 @@ const BookForm = () => {
       <h2>Add New Book</h2>
       <form>
         <input type="text" value={title} onChange={(e) => setTitle(e.target.value)} placeholder="Book Name" />
-        <input type="text" value={author} onChange={(e) => setAuthor(e.target.value)} placeholder="Author" />
-        <select value={category} onChange={(e) => setCategory(e.target.value)}>
+        <select className="action-btn" value={category} onChange={(e) => setCategory(e.target.value)}>
           <option value="Action">Action</option>
           <option value="Fiction">Fiction</option>
           <option value="Non-Fiction">Non-Fiction</option>
         </select>
-        <button type="submit" onClick={addNewBook}>Add Book</button>
+        <button type="submit" className="addbook-btn" onClick={addNewBook}>Add Book</button>
```

Comment:
While making changes based on the previous request, you accidentally added a border that is unnecessary to the button and it makes it look really different from the template. Kindly fix this in order to have a design that is a copy of the original.


<img width="397" alt="image" src="https://github.com/maths29/Book-store/assets/84930355/e7a3384f-a37f-4357-a4a4-0f1b4d2e74da">